doc/source/index.rst

```diff
@@ -240,13 +242,33 @@ which necesitate more resources.
   .. image:: technology_showcase_examples/images/thumb/sphx_glr_tse-001-brake_squeal_thumb.png
     :alt: Brake Squeal Analysis
 
-  :ref:`sphx_glr_ex_0-tecbrakesqueal.py`
+  :ref:`sphx_glr_ex_0-tecbrakesqueal.rst`
```

Comment:
Did you remove the python file?

Reply:
No I didn't. It was already pointing to the rst file but the reference was made with a .py name. I changed the name by it's correct one (a .rst file).